_layouts/documentation.html

```diff
@@ -87,6 +87,11 @@
                                 <li
                                 {% if page.menu == 'gs_dryruns' %} class="nav--vertical__active" {% endif %}>
                                 <a href="/documentation/getstarted/advanced/dryruns">Dry Runs</a></li>
+
+                                <li
+                                {% if page.menu == 'gs_migrationtesting' %} class="nav--vertical__active" {% endif %}>
+                                    <a href="/documentation/getstarted/advanced/migrationtesting">Migration testing</a>
+                                </li>
```

Comment:
Could you put the `<a>` and `</li>` on the same line to be consistent with the other entries

Reply:
62fb2e2